### ISSUE TITLE
Add ability to replace make call

### ISSUE THREAD

**Issue by Mingun** [lpenz/ghaction-cmake#2]
This is needed for Sonar Qube build wrapper: https://docs.sonarqube.org/latest/analysis/languages/cfamily/

instead of
```sh
make VERBOSE=1
```
it requires, that build will run with
```sh
build-wrapper-linux-x86-64 --out-dir build_wrapper_output_directory make VERBOSE=1
```

**Comment by lpenz** [comment]
Interesting
I think getting Sonar Qube to work is a bit more involved than just replacing the `make` call because   the gh action is running inside a container that probably won't have build-wrapper-linux-x86-64 installed, unless we get it there.
Is there a project I can take a look to understand how this works?

**Comment by Mingun** [comment]
Yes, of course. I plan to add action to install it first, as described in the Sonar Qube documentation. Actually, I don't know any working solution, I just trying to setup static analysis checks for https://github.com/AndrewCrewKuznetsov/xneur-devel.
